linker/Linker/LinkContext.cs

```diff
@@ -104,7 +104,10 @@ public bool KeepMembersForDebuggerAttributes
 		public bool IgnoreUnresolved
 		{
 			get { return _ignoreUnresolved; }
-			set { _ignoreUnresolved = value; }
+			set {
+				_ignoreUnresolved = value;
+				_resolver.IgnoreUnresolved = value;
```

Comment:
I don't like this, _resolver could be a shared/global instance and you are changing it under the hood